### DISCUSSION TITLE
Any library or platform open source code we can integrate into Embedded Linux system ?

### DISCUSSION THREAD

**Discussion by Johnson-Liu-Tymphany** [tidal-music/discussions#27] [General]
Is possible to get open source SDK to integrate it into Embedded Linux system? Now there is Amlogic SOC platform at my side,  we want to run Tidal function on that, unlucky, I did not see library or open source for Embedded Linux platform.

**Comment by stefanmoro** [comment]
Hey @Johnson-Liu-Tymphany , unfortunately we don't have any open source SDKs as of today.

We are currently working on opening up some of the SDKs we use internally, but no plans for a linux-native as of today.

**Comment by Johnson-Liu-Tymphany** [comment]
Just to clarify more,  because I see  Tidal App running on phone can discover device and transfer music to Wi-Fi speaker with Linux system ,no clear how it works. Our customer has request to support this feature on audio product, so any advice ？Thanks

**Reply by stefanmoro** [reply]
Hi @Johnson-Liu-Tymphany ,

I believe you are referring to our "[TIDAL Connect](https://tidal.com/connect)" functionality. TIDAL Connect is as of today only available for our so-called device partners.
If you represent a company building a device where you want to integrate TIDAL Connect, you can apply for becoming a device partner at the TIDAL Connect page linked above.
